### DISCUSSION TITLE
Not able to compile for Android

### DISCUSSION THREAD

**Discussion by sarooptech** [axmolengine/axmol#1399] [Q&A] [answered]
I'm getting this error `unable to find library -lpipewire-0.3` when I try to build android studio project. It something related to openal prebuilt.

Found this solution but can't execute those commands without errors.
https://github.com/simdsoft/notes/blob/master/README.md#openal-soft

Dev System : ArchLinux
Target Platform : Android
IDE: Android Studio

**Comment by qacti** [comment]
Same problem here. Any help?

**Reply by sarooptech** [reply]
I don't know. I gave up on it and went back to Cocos2dx 3.17.2 for now. It still works and does the job.

**Reply by halx99** [reply]
I will take a look, it's may easy to solve since the problem caused by cmake find unexpected library for android

**Reply by halx99** [reply]
A fixed PR created: https://github.com/axmolengine/axmol/pull/2200 please check

**Reply by sarooptech** [reply]
I'll try it again.

**Reply by qacti** [reply, marked as answer]
> A fixed PR created: #2200 please check

Now it compiles without errors, thank you :heart:
